tests/runtests.py

```diff
@@ -33,8 +33,8 @@ def check_usable_gpg():
     try:
       subprocess.check_call([gpg, "--version"])
 
-    except OSError as e:
-      pass
```

Comment:
it feels like this change doesn't belong to this commit, does it?

Reply:
Damned, you're right. That must have slipped in while rebasing. Let me see what happened.

Reply:
Thanks for your good eyes, @SantiagoTorres. This indeed slipped in while rebasing onto (a.o.) 23cb4d7e62233b5891e2d7c5d5b6bc07fb1d909d.

Just fixed it.